alert-policies/ray/FreeDiskPercentage.yml

```diff
@@ -0,0 +1,41 @@
+# Name of the alert
+name: Ray Free Disk Percentage
+
+# Description and details
+description: |+
+  This alert is triggered if the free disk percentage lesser than 10% for 5 minutes.
```

Comment:
```suggestion
  This alert is triggered if there is less than 10% of free disk space for 5 minutes.
```

Reply:
Hi @seemantk ,
As per your suggestion, I have made the changes. Could you please review it?